cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -162,10 +162,10 @@ func (cm *certManagerClient) EnsureInstalled() error {
 	// Otherwise install cert manager.
 	// NOTE: this instance of cert-manager will have clusterctl specific annotations that will be used to
 	// manage the lifecycle of all the components.
-	return cm.install()
+	return cm.install(ctx)
 }
 
-func (cm *certManagerClient) install() error {
+func (cm *certManagerClient) install(ctx context.Context) error {
```

Comment:
In the line below, can we get a logger from the context now that we're passing one in? I'm not sure as we're not in a controller.

Reply:
That's fine by me, this PR is already pretty large, and we can consider a follow up if we still want a log from context.